src/types.rs

```diff
@@ -847,7 +852,19 @@ impl Context {
             }
         }
     }
-    /// A human-readable serialization of the Context.
+    /// A human-readable serialization of the `Context`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, parse_context};
+    /// let mut sig = Signature::default();
+    ///
+    /// let context = parse_context(&mut sig, "[!] A CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL)))")
+    ///     .expect("parse of [!] A CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL)))") ;
+    /// 
+    /// assert_eq!(context.pretty(&sig), "[!] A [2, 1, 0]"); 
+    /// ```
```

Comment:
Based on the changes you make to `Context::display`, please update `Context::pretty` to use similar tests so we can highlight the differences.